### ISSUE TITLE
Implement !loadbeforeele (with alias !eleload)

### ISSUE THREAD

**Issue by Yocairo** [opencj-dev/server-gsc#136]
- Find latest save that did not have elevators enabled
- In order to not have GSC spam C commands, better to implement this in C code
- Ensure loading back to this position clears eleOverrideEver flag
- Ensure loading back to another position that allows elevators will have eleOverrideEver flag set

**Comment by iznogod** [comment]
- Latest save that did not have elevators enabled will be the previous one most likely, unless !loadbeforeele value has changed
- wasnt the elevate callback called only once per ele start and ele stop?
- Disagree, if the previous position did have eleoverride enabled it should be safe to load back to it since eles were allowed back then (just load back to whichever position and set the ele-override as it was back then)
- Keep in mind that ele callback is not in sync with serverframes, so the loadposition command will waittillframeend;

**Comment by iznogod** [comment]
Save amount of elevators in player save. Should not be able to !eleload when any elevates (with !ignoreele on) have been done. 
Otherwise we can just load back to the previous save since that should be safe. Previous save in this case is the last LOADED save, not the last MADE save.

**Comment by Yocairo** [comment]
"Should not be able to !eleload when any elevates (with !ignoreele on) have been done."
The point of !eleload is that you can load back to a save that didn't have an ele, so I disagree with this part.

**Comment by iznogod** [comment]
have been done = excluding the current one. If a player bypassed the !eleload earlier, he should not be able to enable !eleload now since the script will have to search quite a few saves to find a non-elevated one.

**Comment by Yocairo** [comment]
Yeah that's the point of eleload, to find the latest save where no ele had been done. If we don't implement that, !eleload is pointless
